docassemble/HousingCodeChecklist/data/questions/customized_screens.yml

```
@@ -12,12 +12,12 @@ include:
 ---
 id: who is using this interview
 question: |
-  Who is using this website?
+  Who are you?
 fields:
-  - I am: person_answering
+  - no label: person_answering
```

```suggestion
  - I am: person_answering
```

`no label` isn't as friendly to screen readers